tools/android/packaging/Makefile.in

```diff
@@ -1,5 +1,8 @@
 include ../../depends/Makefile.include
 
+BUILD_TYPE=@CMAKE_BUILD_TYPE@
+BUILD_TYPE_LC = `echo $(BUILD_TYPE) | tr A-Z a-z`
```

Comment:
BUILD_TYPE_LC := $(shell echo $(BUILD_TYPE) | tr A-Z a-z)
better yet, convert to lc in cmake

Reply:
Convention, or is there something wrong with ``` ` ``` ?

Reply:
kinda, the backticks method is antiquated and we use $(shell) where possible.
Note the := causes the var to be expanded only once at the beginning, so it's preferred here. (also for the line above btw)